squidDrone/tests/i2c/i2c_interface_tests.cpp

```diff
@@ -0,0 +1,139 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "i2c.hpp"
+
```

Comment:
File name still is ambiguous. If i were to look at this folder I would assume you are testing i2c_interface and not i2c. File names shouldn't lie to you.

I understand that you are only testing the functions provided by the interface, but you are testing the specific implementation of i2c. -> i2c_tests

Reply:
Fixed in this commit: https://github.com/elheck/Squiddrone/pull/219/commits/6e2d59d24f9d0aa93cfa99781ebfd1cf39177240